lib/utils.js

```diff
@@ -20,7 +20,14 @@ const labelParser = function (labels) {
 }
 /* Fingerprinting */
 const shortHash = require('short-hash')
-const fingerPrint = function (text, hex) {
+const xxh = require('xxhashjs')
+const fingerPrint = function (text, hex, alg) {
+  alg = alg || process.env.HASH || 'short-hash'
+  switch (alg) {
+    case 'xxhash64':
+      if (hex) return xxh.h64(text, '').toString(16)
+      else return BigInt(xxh.h64(text, '').toString())
```

Comment:
Looks like UINT64 -> String -> BigInt conversion can be replaced with UINT64 -> BigInt:

```suggestion
      if (hex) return xxh.h64(text, '').toString(16)
      else return BigInt(xxh.h64(text, '').digest())
```

Reply:
*fixed

Reply:
Did you see any measurable advantage in terms of performance?

Reply:
not sure if it's measurable, but this extra step is simply not needed (and can slow down things or be about as good in the best case)